NewEvaluation.html

```diff
@@ -0,0 +1,565 @@
+                            <option value="2">2</option>
+                            <option value="3">3</option>
+                        </select>
+                    </li>
+                    <li>Frameworks</li>
+                    <li>
+                        <select name="Frameworks">
+                            <option selected disabled hidden>Evaluation</option>
+                            <option  value="0">0</option>
+                            <option  value="1">1</option>
+                            <option  value="2">2</option>
+                            <option  value="3">3</option>
+                        </select>
+                    </li>
+                </ul>
+                <div class="clearfix"></div>
+            </fieldset>
+
+            
+            <fieldset id="NodeJS">
+                <legend>NodeJS</legend>
+                <ul class="evalForms">
+                    <li>Backend frameworks</li>
+                    <li>
+                        <select name="frameworks">
+                            <option selected disabled hidden>Evaluation</option>
+                            <option  value="0">0</option>
+                            <option  value="1">1</option>
+                            <option  value="2">2</option>
+                            <option  value="3">3</option>
+                        </select>
+                    </li>
+                    <li>Templating</li>
+                    <li>
+                        <select name="Templating">
+                            <option selected disabled hidden>Evaluation</option>
+                            <option  value="0">0</option>
+                            <option  value="1">1</option>
+                            <option  value="2">2</option>
+                            <option  value="3">3</option>
+                        </select>
+                    </li>
+                    <li>DOM Manipulation</li>
+                    <li>
+                        <select name="DOMMan">
+                            <option selected disabled hidden>Evaluation</option>
+                            <option  value="0">0</option>
+                            <option  value="1">1</option>
+                            <option  value="2">2</option>
+                            <option  value="3">3</option>
+                        </select>
+                    </li>
+                    <li>Unit Testing</li>
+                    <li>
+                        <select name="UnitTesting">
+                            <option selected disabled hidden>Evaluation</option>
+                            <option  value="0">0</option>
+                            <option  value="1">1</option>
+                            <option  value="2">2</option>
+                            <option  value="3">3</option> 
+                        </select>
+                    </li>
+                </ul>
+                <div class="clearfix"></div>
```

Comment:
This particular `clearfix` pattern is sooo 2000-whatever-it-was-before-HTML5-appeared! Let's look into more modern options for the same result. ;)